### PR TITLE
adding truncate to dataset

### PR DESCRIPTION
**Pull request by elfenheart**
truncatedataset

### CODE REVIEW

**Comment by nitinmotgi** [comment]
Is it available for streams ?

File: cdap-ui/app/features/datasets/controllers/detail-ctrl.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    var dataSrc = new MyDataSource($scope);
+
+    $scope.truncate = function() {
+      dataSrc.request({
+        _cdapNsPath: '/data/datasets/' + $state.params.datasetId + '/admin/truncate',
+        method: 'POST'
+      }).then(function () {
+        $alert({
+          content: 'Succesfully truncate ' + $state.params.datasetId + ' dataset',
```
Comment (bdmogal) [inline cdap-ui/app/features/datasets/controllers/detail-ctrl.js]: nit: truncated

**Comment by elfenheart** [comment]
@nitinmotgi On a separate PR: #2049

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**